packages/sns/src/sns.wrapper.ts

```diff
@@ -49,6 +54,7 @@ export class SnsWrapper {
   private readonly governance: SnsGovernanceCanister;
   private readonly ledger: SnsLedgerCanister;
   private readonly swap: SnsSwapCanister;
+  private readonly snsIndex: SnsIndexCanister;
```

Comment:
to follow the pattern I would name it `index` instead of `snsIndex` or maybe it's a reserved keyword?

Reply:
It's not reserved, I could use it. But I thought that it might lead to confusion. Not a strong opinion, I can change.

Reply:
just thought it's "governance", "ledger", "swap" etc. no reference to "sns" for the variable that's why I was asking

PR is accepted already, suggestion

Reply:
Yes, I understand. It also thought about it. I wasn't sure one or the other. I renamed to `index`, makes sense to me as well.

I just wish they would have call it something else, haha.